api/core/model_runtime/model_providers/sagemaker/tts/tts.py

```diff
@@ -77,7 +77,7 @@ def validate_credentials(self, model: str, credentials: dict) -> None:
         """
         pass
 
-    def _detect_lang_code(self, content: str, map_dict: dict = None):
+    def _detect_lang_code(self, content: str, map_dict: Optional[dict] = None):
```

Comment:
This parameter is unused, you can remove it.

Reply:
(ditto) Will remove the unused parameter in the follow-up PR, covering the method's signature and usages.